api/v1/flow/flow.proto

```diff
@@ -156,6 +160,22 @@ message Layer7 {
     }
 }
 
+// TraceContext contains trace context propagation data, ie information about a
+// distributed trace.
+// For more information about trace context, check the W3C Trace Context
+// specification: https://www.w3.org/TR/trace-context/
+message TraceContext {
+    // parent identifies the incoming request in a tracing system.
+    TraceParent parent = 1;
+}
+
+// TraceParent identifies the incoming request in a tracing system.
+message TraceParent {
```

Comment:
Now why `TraceParent` is a separate message? Is it basically following the parent/state split from the standard?

Reply:
Yes (see comment [here](https://github.com/cilium/cilium/pull/21456#discussion_r983280072) for the rationale).